frontend/src/main/scala/bloop/cli/completion/Case.scala

```diff
@@ -0,0 +1,22 @@
+package bloop.cli.completion
+
+import caseapp.core.Arg
+import caseapp.Name
+
+object Case {
+  private val Kebab = "-([a-z])".r
+  private val Camel = "([A-Z])".r
+
+  private def camelToKebab(camel: String): String = {
+    val m = Camel.pattern.matcher(camel)
+    val sb = new StringBuffer
+    while (m.find()) {
+      m.appendReplacement(sb, "-" + m.group().toLowerCase())
+    }
+    m.appendTail(sb)
+    sb.toString
+  }
+
+  def kebabizeArg(arg: Arg): Arg = arg.copy(extraNames =
```

Comment:
Can we put the body of this method inside curly braces and move the implementation of `extraNames` to a new `val`? :smile:

Reply:
Sure!